config/text_metrics_keywords.yml

```diff
@@ -0,0 +1,14 @@
+development:
```

Comment:
IMHO, these keywords could be the same in different environments. Hence, I think you do not need to duplicate them twice.